getting-started/gcp/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+# Variable value assignments go here
```

Comment:
It's poor form to check `terraform.tfvars` into a repository, even if part of a beginner demo. Do a `git rm` or `git mv terraform.tfvars terraform.tfvars.example` on this file and add `terraform.tfvars` to `.gitignore`.

Reply:
Good suggestion, I'll do that.